templates/addons/machine-health-check/machinehealthcheck.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
do you think this stuff is standard enough we should just add it to the default flavor? or could it be disruptive enough to keep it as an opt-in in a separate flavor?

Reply:
I believe it should be opt-in rather than included by default since it introduces extra complexity that may not be desired